inclusive-design/wecount.inclusivedesign.ca
# fix: apply the new design to home page tiles (resolves #421)

## Description

Apply the new design to home page tiles. The new design can be found at [the Figma](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=1622%3A4592).

## Steps to test

1. Go to the home page;
2. The style of tiles should match the design;
3. Apply UIO contrast themes;
4. Home tile background images and colors should disappear when a contrast theme is applied.

**Expected behavior:** 

See above.
